js/volunteer-badges.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
```

Comment:
This seemed like a useful way of adding badges rather than maintaining all the markup. I almost considered laying out the bios with jquery too since there are so many of them, but it would be a very sad site if someone had javascript disabled.